server/container/guice/blob-objectstorage-guice/src/main/java/org/apache/james/modules/objectstorage/AwsS3ConfigurationReader.java

```diff
@@ -0,0 +1,60 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.modules.objectstorage;
+
+import java.net.URI;
+
+import org.apache.commons.configuration.Configuration;
+
+import com.google.common.base.Strings;
+import com.google.common.base.Preconditions;
+
+public class AwsS3ConfigurationReader {
+
+    static final String OBJECTSTORAGE_ACCESKEYID =
```

Comment:
please don't duplicate `objectstorage` prefix as it's used in different classes (far from each others)